dll/cpl/mmsys/lang/ro-RO.rc

```diff
@@ -123,7 +123,7 @@ BEGIN
     ICON 4379, -1, 7, 8, 20, 20
     LTEXT "Alegeți modul de așezare a difuzoarelor conectate la acest calculator.", -1, 36, 8, 215, 27
     CONTROL 110, 5376, "STATIC", SS_BITMAP | SS_CENTERIMAGE, 37, 40, 209, 112, WS_EX_TRANSPARENT
-    LTEXT "Amplasament:", 8502, 53, 169, 55, 10
+    LTEXT "Configurarea difuzoarelor:", 8502, 53, 169, 55, 10
```

Comment:
This text is way too long to fit the control of 55 units width. Do a screenshot to see it yourself!